### PR TITLE
resolving bug in PTDF with lines and transformers

### PR DESCRIPTION
**Pull request by claytonpbarrows**
This PR addresses a bug where the :network_flow constraint container was being created for every branch device type, overwriting previous instances of the container.

### CODE REVIEW

**Comment by claytonpbarrows** [comment]
This still results in a situation where you can have leftover `undef` valuues in the constraint container if your template fails to cover every `ACBranch` device in the network.

**Comment by claytonpbarrows** [comment]
@jd-lara this should be ready now

**Comment by claytonpbarrows** [comment]
closes #644

File: src/network_models/powermodels_interface.jl [hunk 1]
```diff
@@ -415,8 +412,8 @@ function PMvarmap(system_formulation::Type{S}) where {S <: PM.AbstractPowerModel
     )
     pm_var_map[PSY.DCBranch] = Dict(
         :p_dc => (
-            from_to = FlowActivePowerFromToVariable,
-            to_from = FlowActivePowerToFromVariable,
+            from_to = FlowActivePowerVariable,#FlowActivePowerFromToVariable,
+            to_from = nothing,#FlowActivePowerToFromVariable,
```
Comment (jd-lara) [inline src/network_models/powermodels_interface.jl:416]: do we need to keep this nothing here?

**Review by jd-lara — APPROVED** [review]
Looks good, this is a great improvement. There are some minor comments to remove commented-out code.

**Review by jd-lara — APPROVED** [review]

**Pull request merged**